### PR TITLE
Refactor classes into files

### PR DESCRIPTION
**Pull request by sciencemanx**
This makes the code easier to read and navigate through! Thanks.

### CODE REVIEW

**Comment by sciencemanx** [comment]
@kerrickstaley Bump 😄

File: genanki/deck.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (kerrickstaley) [inline genanki/deck.py]: Please remove newline before merging.

File: genanki/model.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
+
```
Comment (kerrickstaley) [inline genanki/model.py]: Please remove newline before merging.

File: genanki/note.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (kerrickstaley) [inline genanki/note.py]: Please remove newline before merging.

File: genanki/package.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (kerrickstaley) [inline genanki/package.py]: Please remove newline before merging.

File: genanki/util.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (kerrickstaley) [inline genanki/util.py]: Please remove newline before merging.

**Comment by kerrickstaley** [comment]
Re: previous review, what I meant was: Please fix the newlines, and then I will merge your PR.

**Comment by sciencemanx** [comment]
Sounds good.

**Comment by sciencemanx** [comment]
@kerrickstaley Done!

**Comment by kerrickstaley** [comment]
Thanks! I'll review your last PR soon.

**Pull request merged**